CLAUDE.md

```
@@ -0,0 +1,189 @@
+| `docs/*` | Documentation | `develop` |
+
+### Pull Request Rules
+
+1. **All PRs must target `develop` branch** (not `main`)
+2. PRs to `main` are only for releases from `develop`
+3. Require code review before merging
+4. All CI checks must pass
+
+### Commit Message Format
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)